jsonifier/README.rst

```diff
@@ -192,8 +192,45 @@ new requirements could be met without needing to add field or change any code.
 Note that the export values with the simple parser depends on the record's lang;
 this is in contrast with full parsers which are designed to be language agnostic.
 
+default_function parameter
+==========================
 
-NOTE: this module was named `base_jsonify` till version 14.0.1.5.0.
+In order to facilitate the usage of the many options jsonifier has, there is the 
+default_function=True options we can pass to rec.jsonify()
+this will impose as function  for all fields default_function, in order to shorten
+parser expression.
+
+default function currently retuns field: {fieldname: string_of_field, value: value} instead of
+the standard  field: value.
+
+It can be overwritten per-model or globally directly on models.py
+
+
+   Examples of default_function usage:
+
+.. code-block:: python
+   
+    # example 1 
+    parser = [('name')]
+    a.jsonify(parser=parser)
+    [{'name': 'SO3996'}]
+    >>> a.jsonify(parser=parser, default_function=False)
+    [{'name': 'SO3996'}]
+    >>> a.jsonify(parser=parser, default_function=True)
```

Comment:
TBH I find the parameter name `default_function` confusing. I can understand where you get it from, but to the end user it suggests that some kind of default would be applied, whereas that's not what it does, what it does is to add field names to the output. So `with_field_names=True` would be more intuitive.

Reply:
I put it as "default function" because that solution could had been used for anything and kept a generic name.
The default was adding the fieldnames. but it could had been something else.
that solution was thrashed and now is with_fieldnames as it should be.